plugin-worldclock/lxqtworldclockconfigurationtimezones.cpp

```diff
@@ -119,7 +119,7 @@ int LXQtWorldClockConfigurationTimeZones::updateAndExec()
         if (qStrings.size() == 1)
             qStrings.prepend(tr("Other"));
 
-        QTreeWidgetItem *tzItem = new QTreeWidgetItem(QStringList() << qStrings[qStrings.length() - 1] << timeZone.displayName(now) << timeZone.comment() << QLocale::countryToString(timeZone.country()));
+        QTreeWidgetItem *tzItem = new QTreeWidgetItem(QStringList() << qStrings[qStrings.length() - 1] << timeZone.displayName(now) << timeZone.comment() << QLocale::territoryToString(timeZone.territory()));
```

Comment:
My understanding is that failing localization and timezone settings are caused by this change to `territory`.
Deprecated:

>Returns a [QString](https://doc.qt.io/qt-6/qstring.html) containing the name of country.

> Returns the two-letter territory code for territory, as defined in the ISO 3166 standard.